streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -103,6 +103,7 @@
     public static final String VALUE_SERDE_CLASS_CONFIG = "value.serde";
     public static final String VALUE_SERDE_CLASS_DOC = "Serializer / deserializer class for value that implements the <code>Serde</code> interface.";
 
+    /**<code>user.endpoint</code> */
     public static final String USER_ENDPOINT_CONFIG = "user.endpoint";
     public static final String USER_ENDPOINT_DOC = "A user defined endpoint that can be used to connect to remote KafkaStreams instances. Should be in the format host:port";
```

Comment:
I think we should stick with host:port for the near term unless there is specific requests in the KIP feedback.